doc/_release/README.md

```diff
@@ -1,4 +1,14 @@
-# How to release Spoon?
+# Spoon Releases
+
+## Versioning
+
+Spoon uses a three digit version number MAJOR.MINOR.HOTPATCH
+
+* we bump MAJOR when there is a big new feature or a strongly breaking change. We use it mostly for new features since we take special care to backward compatibility.
+* we bump HOTPATCH when a normal release is not operational.
```

Comment:
Maybe change the order of the points to follow the order above `MAJOR.MINOR.HOTPATCH` ?